quodlibet/po/fr.po

```diff
@@ -426,9 +419,8 @@ msgid "Merge"
 msgstr ""
 
 #: ../quodlibet/browsers/collection/prefs.py:188
-#, fuzzy
 msgid "Album Collection Preferences"
-msgstr "Préférences de l'affichage liste d'albums"
+msgstr "Préférences de la collection d'album"
```

Comment:
I would write "albums", as "collection de" or "liste de" will always be followed by a plural.